pyproject.toml

```diff
@@ -92,14 +91,12 @@ help = "Export installed packages in requirements.txt format"
 [tool.taskipy.tasks]
 black = { cmd = "black --check .", help = "dry run of black" }
 codecov-validate = { cmd = "curl --data-binary @.codecov.yml https://codecov.io/validate", help = "Validate `.codecov.yml` with their api." }
-pre_cov-server = "coverage html"
-cov-server = { cmd = "python -m http.server 8012 --bind 127.0.0.1 --directory htmlcov", help = "Start an http.server for viewing coverage data."}
+cov-server = { cmd = "coverage html", help = "Start an http.server for viewing coverage data."}
+post-cov-server = "python -m http.server 8012 --bind 127.0.0.1 --directory htmlcov"
 docs = { cmd = "mkdocs serve", help = "Run the docs on a local automatically reloading server"}
 flake8 = { cmd = "python -m flake8", help = "Lints code with flake8" }
 lint = { cmd = "pre-commit run --all-files", help = "Checks all files for CI errors" }
 precommit = { cmd = "pre-commit install --install-hooks", help = "Installs the precommit hook" }
-pytest-docs = { cmd = "pytest --no-cov --co -qqq --docs tests/docs.md", help = "Create docs for tests using pytest-docs." }
 report = { cmd = "coverage report", help = "Show coverage report from previously run tests." }
 start = { cmd = "python -m modmail", help = "Run bot" }
-test = { cmd = "pytest -n auto --dist loadfile --cov-report= --cov= --docs tests/docs.md", help = "Runs tests and save results to a coverage report" }
-test-nocov = { cmd = "pytest --no-cov", help = "Runs tests without creating a coverage report" }
+test = { cmd = "pytest -n auto --dist loadfile", help = "Runs tests and save results to a coverage report" }
```

Comment:
A bit of this PR, but could you break this down into "groups" so that it is easier to view the tasks.